src/entries-list/entries-list.ts

```diff
@@ -0,0 +1,178 @@
+import { LitElement, html, css } from "lit";
+import { customElement, state } from "lit/decorators.js";
+import {
+  getDashboardEntries,
+  ImportableEntry,
+  ListEntriesResult,
+} from "../api/list-entries";
+import { openWizardDialog } from "../wizard";
+import "@material/mwc-button";
+import "../components/esphome-button-menu";
+import "../components/esphome-card";
+import { subscribeOnlineStatus } from "../online_status";
+import "./node-card";
+import "./import-card";
+
+@customElement("esphome-entries-list")
+class ESPHomeEntriesList extends LitElement {
+  @state() private _entries?: ListEntriesResult;
+  @state() private _onlineStatus?: Record<string, boolean>;
+  @state() private _highlightedName?: string;
+
+  private _updateEntriesInterval?: number;
+  private _nodeStatusUnsub?: () => void;
+
+  protected render() {
+    if (this._entries === undefined) {
+      return html``;
+    }
+    if (
+      this._entries.configured.length === 0 &&
+      this._entries.importable.length === 0
+    ) {
+      return html`
+        <div class="welcome-container">
+          <h5>Welcome to ESPHome</h5>
+          <p>It looks like you don't yet have any nodes.</p>
```

Comment:
is it a node or an entry? Our API and components call it `entry`. If we still can, let's try to be consistent.

Reply:
Right, I guess `device` makes the most sense - we already consistently use "device" in the rest of the UI when talking about the thing running esphome.

I'll rename the APIs/components to call it `device`